### DISCUSSION TITLE
We want a chat box on github

### DISCUSSION THREAD

**Discussion by cracker911181** [community/community#13468] [Other Feature Feedback, Questions, & Ideas]
We want a "Chat-Box" like facebook. We often have problems with programming. If there was a chat box, then we could solve it by chatting with each other. I hope you will think about this.

**Comment by suprstarrd** [comment]
I'm not sure I'm getting what you're trying to say, but if you mean a public chatbox that appears on someone's profile then please no! That sounds like a recipe for disaster

**Reply by djjohnsongeek** [reply]
Yeah, no thanks. If I want to chat I'll use something else like slack.
